app/javascript/packs/components/RemoveMultiple.js

```diff
@@ -41,6 +43,16 @@ function removeMultiple(id) {
   updateCount();
 }
 
+function showButton() {
+  const button = document.querySelector('button[data-multiple-add]')
+  button.classList.remove('hidden')
+}
+
+function removePTag() {
+  const p = document.getElementById('multiple-respondent-p')
```

Comment:
There is a much easier way - mark anything in the html with data: { 'multiple-add': true } that needs hiding and showing.  Add the hidden class in the html where needed

Then, change the showButton function and name it showHiddenElements - then change the css selector to be '*[data-multiple-add]'. - you then won't need the removePTag function

Reply:
You will also need to change the document.querySelectorAll and loop over its results and remove the hidden class from all elements found